### PR TITLE
Update the disaster recovery page to address feedback

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 1]
```diff
@@ -173,7 +175,7 @@ SHOW SERVERS;
 ==== Path to correct state
 Use the following steps to remove lost servers and add new ones to the cluster.
 To remove lost servers, any allocations they were hosting must be moved to available servers in the cluster.
-This can be done in two different ways:
+This is done in two different ways:
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:178]:
```suggestion
This is done in two different steps:
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2026.surge.sh

Updated pages:
- [Disaster recovery](https://neo4j-docs-operations-2026.surge.sh/operations-manual/2025.01/clustering/disaster-recovery)

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 1 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:178]: I also like the suggestion with "steps".

**Review by AnnaSjerling — COMMENTED** [review]
There is also one place with "write-enabled" which I think we should change to "write-available" under make system database write available -> objective. 

Also, in the note where we explain the terms we have written "write-available" and "write unavailable". I don't know which is "correct", with or without "-" but maybe they should be the same?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2026.surge.sh

Updated pages:
- [Disaster recovery](https://neo4j-docs-operations-2026.surge.sh/operations-manual/2025.01/clustering/disaster-recovery)

**Review by AnnaSjerling — APPROVED** [review]
Nice! We are done, yey!

**Pull request merged**